api/groups/proofGroup.go

```diff
@@ -137,21 +221,41 @@ func (pg *proofGroup) getProof(c *gin.Context) {
 		return
 	}
 
-	hexProof := make([]string, 0)
-	for _, byteProof := range proof {
-		hexProof = append(hexProof, hex.EncodeToString(byteProof))
+	var mainProofHex, dataTrieProofHex []string
+	var dataTrieRootHash, dataTrieValue string
+	if mainTrieResponse != nil {
+		mainProofHex = bytesToHex(mainTrieResponse.Proof)
+	}
+	if dataTrieResponse != nil {
+		dataTrieProofHex = bytesToHex(dataTrieResponse.Proof)
+		dataTrieRootHash = dataTrieResponse.RootHash
+		dataTrieValue = hex.EncodeToString(dataTrieResponse.Value)
 	}
 
 	c.JSON(
 		http.StatusOK,
 		shared.GenericAPIResponse{
-			Data:  gin.H{"proof": hexProof},
+			Data: gin.H{
+				"mainProof":        mainProofHex,
```

Comment:
Should have been:
```
{
 "data": {
     "proofs": {  
           "mainProof": ..
            ....
      }
 }
....
```
you placed the attributes right under the `data` field which doesn't follow the format of other endpoints

Reply:
Done.